### DISCUSSION TITLE
Not sure if it's installing correctly

### DISCUSSION THREAD

**Discussion by npcombs1992** [synman/Ender-3-S1-Pro-Firmware#19] [Q&A]
Hi there,

Thank you for all your hard work on putting this together! I've been working with 3D printers now for several years, but this is my first attempt at installing a 3rd party software.  I thought I was following your steps and it appears the new software got installed, but none of the UI changed and I wasn't sure if there was something I missed. I recorded a video showing my steps, it's about 2 minutes long but shows the process from start to finish. If you could take a look and let me know what I missed, I would definitely appreciate it!

[https://photos.app.goo.gl/GsavaypryZkf6oEM8](https://photos.app.goo.gl/GsavaypryZkf6oEM8)

Thans again for your help!

**Comment by synman** [comment]
Yup, you did everything as you should and I see the new version on the LCD.   You are not going to see any UI changes, at least not yet.  We are currently stuck using the UI Creality provides but I am able to change the way the machine responds it.

You grabbed the UBL build.  It's a couple builds back (I recommend grabbing the latest as there have been a couple bug fixes and features added).  All of the commands necessary to build UBL mesh are integrated with the UI but it's a good idea for you to learn more about it if you've never used it before -- there's a discussion thread I created for it, check It out if so.

Same goes for a few of the other capabilities such as Input Shaping, Linear Acceleration, Advanced Pause (Filament changes with senders like Octoprint), etc.
